docs/README-he.md

```diff
@@ -0,0 +1,190 @@
+
+<div dir="rtl">
+תוכנה להזרמת שירים מבוססת על מקורות חינמיים
+
+![Showcase](https://i.imgur.com/G9BqIHl.png)
+
+# קישורים
+
+[האתר הרשמי](https://nuclear.js.org)
+
+[Mastodon](https://mstdn.io/@nuclear)
+
+[טוויטר](https://twitter.com/nuclear_player)
+
+ערוץ תמיכה (Matrix): `#nuclear:matrix.org`
+
+ערוץ הדיסקורד: https://discord.gg/JqPjKxE
+
+תרגומים נוספים לדף:
+
+<kbd>[<img title="Português" alt="Português" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/br.svg" width="22">](README-ptbr.md)</kbd>
+<kbd>[<img title="Svenska" alt="Svenska" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/se.svg" width="22">](README-se.md)</kbd>
+<kbd>[<img title="English" alt="English" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/us.svg" width="22">](README.md)</kbd>
+<kbd>[<img title="Hebrew" alt="Hebrew" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/il.svg" width="22">](README-he.md)</kbd>
+<kbd>[<img title="Italiano" alt="Italiano" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/it.svg" width="22">](README-it.md)</kbd>
+
+## מה זה בכלל Nuclear?
+Nuclear זוהי תוכנה חינמית להזרמת תכנים ממקורות חינמיים ברחבי האינטרנט.
+
+מבינכם שמכירים את [mps-youtube](https://github.com/mps-youtube/mps-youtube), זו תוכנה דומה אך עם תוספת של ממשק משתמש גרפי. אנחנו מתמקדים בעיקר על השמע. דמיינו על ספוטיפיי אבל אתם לא צריכים לשלם עבורו ועם ספרייה גדולה יותר.
+
+## אנחנו משתמשים בElectron, אם אתם נגד
+קראו [כאן](electron.md).
+
+## פיצ׳רים
+
+- אפשרות לחיפוש ולשמוע שירים הישר מהאתר YouTube (כולל קישור עם פלייליסטים), Jamendo, Audius ו - SoundCloud
+- אפשרות חיפוש לאלבומים (אשר מופעלים על ידי Last.fm ו - Discogs), חיפוש שירים על פי שם האמן ושם השיר (בתהליך, יכול להיות באגי לפעמים)
+- אפשרות ליצירת תור שמיעת שירים, שגם יכולה לייצא כפלייליסט
+- אפשרות טעינת פלייליסטים שמורים (שמורים כקובץ JSON)
+- אפשרות לקשר את החשבון שלך בLast.fm (בנוסף גם מאפשר להצגת סטטוס ׳מושמע כרגע...׳)
+- העדכונים הכי חדשים כולל ביקורת - שירים ואלבומים
+- אפשרות חיפוש לפי סגנון מוזיקה
+- אפשרות למצב רדיו (דומה לתור שירים אך עם שירים דומים למה שמושמע)
+- אפשרות לאינסוף הורדות (מופעל על ידי YouTube)
+- אפשרות לתצוגת מילות השירים בזמן אמת
+- אפשרות חיפוש על פי פופולריות
+- אפשרות יצירת רשימה של שירים מועדפים
+- אפשרות שמיעה של תכנים מתוך המחשב
+- אין צורך במשתמש
+- אין שום פרסומות
+- אין צורך בלחתום על שום דבר כדי לתרום לפרוייקט
+
+## מידע למשתמש
+https://nuclearmusic.rtfd.io/
+
+## חבילות שנתמכות על ידי הקהילה
+
+לפניכם רשימה של חבילות בשביל שימוש שונים, רובם נתמכות על ידי אדם צד שלישי. אנחנו רוצים להודות לתומכים על עבודתם.
```

Comment:
I would phrase this differently, to emphasize the use for different package managers. Something like:
`לפניכם רשימה של החבילות עבור מנהלי החבילות השונים, רובן מתוחזקות על ידי תומכי צד שלישי`